### DISCUSSION TITLE
Migrating from password login to OIDC

### DISCUSSION THREAD

**Discussion by dereulenspiegel** [immich-app/immich#8045] [Q&A] [answered]
I am running Immich for quite some and really like it. So far I have been using it with normal password based logins with internal immich users. But the latest Authelia release (4.38) now allows to give specific user groups access to specific OIDC clients. With this in mind I would now like to migrate my existing immich installation to OIDC authentication.
Is this possible at all? And if so how do enable one user as the admin user?

Thanks in advance :)

**Comment by schuhbacca** [comment, marked as answer]
Existing users will just be tied by email address. So as long as they match between authelia and immich you shouldn't really have to do anything.

**Reply by dereulenspiegel** [reply]
Thanks for the super fast answer. I will try this out and report back :)

**Comment by dereulenspiegel** [comment]
Finally got around to trying this out. And it works as described. My existing users had the exact same email addresses as the oidc users, so everything just mapped. My admin user also stayed and admin user.
Thanks again :)
